src/NuGet.Clients/NuGet.PackageManagement.UI/Options/SourceMappingViewModel.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class SourceMappingViewModel
+    {
+        public string ID { get; private set; }
+        public IReadOnlyList<PackageSourceContextInfo> Sources { get; private set; }
+
+        public string SourcesString
+        {
+            get
+            {
+                return string.Join(", ", Sources.Select(s => s.Name));
+            }
+        }
```

Comment:
nit: use compact syntax:

```suggestion
        public string SourcesString => string.Join(", ", Sources.Select(s => s.Name));
```